accepted/2025/rid-specific-tool-packages.md

```diff
@@ -0,0 +1,91 @@
+# RID-Specific .NET Tool Packages
+
+**Owner** [Daniel Plaisted](https://github.com/dsplaisted)
+
+## Summary
+
+This proposal adds the ability for a .NET Tool to have separate packages for each supported OS and architecture.
+
+In .NET, the combination of OS and architecture is represented by a Runtime Identifier (RID).  Today, .NET Tools support native assets, but the native assets for all of the supported Runtime Identifiers need to be included in the same package.  For tools with large native dependencies, or if the entire tool is native (for example via NativeAOT), this multiplies the package download size by the number of supported RIDs.
```

Comment:
The exact same problem exists for regular nuget packages too. Any chance we can generalize the solution to solve the problem for all types of nuget packages, and not just tools? 

Related question: Is the downloader for the proposed dependent packages going to be implemented in nuget or in the SDK?

Reply:
https://github.com/NuGet/Home/issues/10571 is an issue about the problem for regular nuget packages